jlm/util/Makefile.sub

```diff
@@ -19,7 +19,7 @@ libutil_HEADERS = \
     jlm/util/intrusive-list.hpp \
     jlm/util/iterator_range.hpp \
     jlm/util/Math.hpp \
-    jlm/util/PtrIterator.hpp \
+    jlm/util/IteratorWrapper.hpp \
```

Comment:
The files seem to be alphabetically ordered in this list. It is currently at the wrong position